Read-and-save-PowerPoint-presentation/Open-and-save-PowerPoint/Google-Drive/Save-PowerPoint-document/Save-PowerPoint-document/Program.cs

```diff
@@ -0,0 +1,109 @@
+﻿using Google.Apis.Auth.OAuth2;
+using Google.Apis.Drive.v3;
+using Google.Apis.Services;
+using Google.Apis.Util.Store;
+using File = Google.Apis.Drive.v3.Data.File;
+using Syncfusion.Presentation;
+
+namespace Save_PowerPoint_document
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            //Create a new instance of PowerPoint Presentation file
```

Comment:
Please add pull stop at the end of the comments

Reply:
Added properly